tests/RegExpNode.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from './helper';
+import RegExpNode from '../src/RegExpNode';
+
+describe('RegExpNode', () => {
+    it('should create RegExpNode instance with string pattern', () => {
+        const aaORbb = new RegExpNode('^aa|bb', true, 'data');
+
+        expect(aaORbb).to.have.property('str', '^aa|bb');
+        expect(aaORbb).to.have.property('regExp');
+        // @ts-ignore
```

Comment:
What's wrong here? Does `tsc` throw an error, so we need to shut it up?

Reply:
regExp is private